.gitignore

```diff
@@ -97,6 +97,7 @@ dist
 # Comment in the public line in if your project uses Gatsby and not Next.js
 # https://nextjs.org/blog/next-9-1#public-directory-support
 # public
+# public
```

Comment:
what is this?

Reply:
The 'idea' folder? It is PyCharm's own folder that it uses to save its own configuration.